### PR TITLE
Jedd/empty character fix

### PR DESCRIPTION
**Pull request by JeddSJr**

### CODE REVIEW

File: src/components/VolunteersForm.js [hunk 1]
```diff
@@ -15,14 +15,22 @@ export default class VolunteersForm extends React.Component {
...
+      this.state = {
         minimumSelectionInterest: false,
-        minimumSelectionHAU: false
+        minimumSelectionHAU: false,
       };
     }
 
     handleOnChange = (event) => {
-      this.setState({ [event.target.name]: event.target.value });
+      if(trim(event.target.value) != ""){
```
Comment (kandloic) [inline src/components/VolunteersForm.js:25]: Use the `!==` operator instead

File: src/components/VolunteersForm.js [hunk 2]
```diff
@@ -15,14 +15,22 @@ export default class VolunteersForm extends React.Component {
...
+        document.getElementById(event.target.id).setCustomValidity("");
```
Comment (kandloic) [inline src/components/VolunteersForm.js:26]: Is this line necessary?

**Review by kandloic — CHANGES_REQUESTED** [review]
Overall looks good! A few small changes and then you should be good to go!

File: src/components/VolunteersForm.js (hunk 2 above)
Comment (JeddSJr) [reply src/components/VolunteersForm.js:26]: Line 26 ? Yes, absolutely. Without it the form will not check again if the input is valid and always return an error tooltip

Comment (kandloic) [reply src/components/VolunteersForm.js:26]: Ah, gotcha!

**Review by kandloic — APPROVED** [review]

**Pull request merged**